outline.md

```diff
@@ -0,0 +1,44 @@
+# Learning Outline 
+
+## Content
+
+The workshop is divided into the following three sections:
+
+### The Unix Shell
+
+The aim of this section is not to teach learners basic unix commands,
+or syntax for bash scripting,
+but rather to motivate them
+to **automate tasks and develop pipelines**.
+Topics covered will include filesystem navigation and manipulation,
+redirection and piping of standard input and output,
+file permissions, and shell scripts.
+
+### Cluster structure and scheduling
+
+This section will be very brief. It is meant to convey a simplistic mental model of the cluster,
+and how tasks get submitted, assigned and executed on the cluster. The used HPC infrastructure
+
+At the end of the section, they will
+submit a number of "Hello World" style batch jobs that are aligned to the Unix Shell section above.
+
+### Parallel workflows
+
+This section will guide learners through the process of
+performing a large HPC simulation (code available),
```

Comment:
On our systems (at Juelich Supercomputing Centre) we provide both Python and the SciPy stack as modules so this wouldn't work for us without touching modules. 
There are 3 variants (Tcl, C, Lua) that I know of but all of them share the same basic commands (`module avail`, `module load `, `module list`) that would cover what we need.
There are many variants of resource manager too and I think we don't have much choice but to provide the submission files in all variants. Unfortunately this is the problem with the HPC space, the heterogeneous environments make it hard to create homogeneous tutorials (without baking in some additional cleverness to handle this...or you provide the training cluster).